python_benchmarks/21_polars_streaming.py

```diff
@@ -1,20 +1,17 @@
-import polars
+import polars as pl
 
 
 def polars_streaming(fname):
-    dtypes = {f"column{i}": (polars.UInt32 if i == 0 else polars.Float32())
+    dtypes = {f"column{i}": (pl.UInt32 if i == 0 else pl.Float32())
               for i in range(9)}
-    return (polars.scan_csv(fname, dtypes=dtypes)
-                  .select(
-                      polars.sum_horizontal(polars.all()
-                                                  .exclude("0")
-                                                  .pow(2)
-                      ).sqrt()
-                       .alias("sum_sq"))
-                  .filter(polars.col("sum_sq") < 1e16)
-                  .count()
-                  .collect(streaming=True)
-    )
+    return (pl.scan_csv(fname, dtypes=dtypes)
+            .filter(pl.sum_horizontal(pl.all()
+                                      .exclude("column0")
```

Comment:
Column `0` doesn't exist so Polars did more work than other solutions here as it also included the id column